src/components/organisms/Nav.js

```diff
@@ -21,7 +25,13 @@ const Nav = () => {
   } = useAuthContext();
   const [modalState, setModalState] = useState(modalStates.closed);
   const [isAddingStory, setIsAddingStory] = useState(false);
-
+  const [isDisabled] = useState(true);
+  const [Username, setUsername] = useState('');
```

Comment:
Why is this capitalized? To stick with correct casing, this should be camelCased. Same goes for Email and Password